src/main/scala/uk/gov/nationalarchives/api/update/ApiUpdate.scala

```diff
@@ -11,12 +11,11 @@ import scala.concurrent.{ExecutionContext, Future}
 import scala.language.postfixOps
```

Comment:
Unused import: `import com.typesafe.config.ConfigFactory`

Reply:
I've taken this out.